mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -255,6 +255,7 @@ def bind(self, bus):
             self.event_scheduler.set_id(self.skill_id)
             self._enclosure = EnclosureAPI(bus, self.name)
             self._register_system_event_handlers()
+
```

Comment:
This is nitpicky, but can we remove this so the file doesn't get touched by this PR?